### PR TITLE
Template for new Docs pages

### PR DESCRIPTION
**Pull request by samodell**
Fixes #issue-number

## Proposed Changes

- Adds a template with some guide text for creating a new page.
- Uses comments to explain how to fill out frontmatter

### CODE REVIEW

**Comment by samodell** [comment]
FYI @RichieEscarez , after we are happy with the template, I'll open a PR in the Community repo to add a link to the this template from the Docs Contributing guide: https://github.com/knative/community/blob/master/DOCS-CONTRIBUTING.md

File: contributing/new-page-template.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+# This section is called the "frontmatter" for your page
+title: "Title For Your Page" # Use title case for titles
```
Comment (RichieEscarez) [inline contributing/new-page-template.md]: Should be sentence capitalization for all titles and sections

File: contributing/new-page-template.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#linkTitle: "Link For This Page in the Sidebar"
+# The linkTitle field (above) is optional; use it to provide a shorter link if your page title is very long
+weight: 01 # This affects the placement of the link in the sidebar on the left. Pages are ordered from top to bottom by weight.
```
Comment (RichieEscarez) [inline contributing/new-page-template.md]: Should start and go by 10s to allow insertion of future topics. Maybe also clarify that these are relative to the weight of other files?

File: contributing/new-page-template.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- A Kubernetes cluster with [Knative installed](../install/README.md). <!-- Update this relative link as needed,
+depending on where the new page is located in the file structure. -->
+- Anything else?
+
+## Break steps into logical sections
+
+Avoid nesting headings directly on top of each other with no text inbetween.
+
+1. Use ordered lists for steps.
```
Comment (RichieEscarez) [inline contributing/new-page-template.md]: Include a nested code example and highlight the importance of using spaces for Indentation (not tabs)

**Comment by samodell** [comment]
@RichieEscarez  I think I addressed your feedback!

File: contributing/new-page-template.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+# This section is called the "frontmatter" for your page
+title: "Title for your page" # Use sentence case for titles
+#linkTitle: "Link for this page in the sidebar"
+# The linkTitle field (above) is optional; use it to provide a shorter link if your page title is very long
+weight: 10 # This affects the placement of the link in the sidebar on the left. Pages are ordered from top to bottom by weight, lowest to highest.
+type: "docs" # You won't need to update this.
+aliases:
```
Comment (RichieEscarez) [inline contributing/new-page-template.md]: also indicate that its optional
```suggestion
#aliases:
```

File: contributing/new-page-template.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  - # Has the page ever moved? If yes, include the prior location here, starting with /docs/, and the old URL will redirect to the new location. For a new page, there should be no aliases.
```
Comment (RichieEscarez) [inline contributing/new-page-template.md]:
```suggestion
#  - /docs/example/redirect/moved-renamed-page
#  - /docs/another-example
# Has the page ever moved? If yes, include the prior location above, starting with path from the site root (for example /docs/, /blog/, or /community/). The old URL will redirect to this new file. For a new pages, "aliases" are not required.
```

**Comment by samodell** [comment]
@RichieEscarez  If we want to add that info, we could later! But I think this might be good as-is for a new page template?

**Pull request merged**